source/routing/tests/raptor_test.cpp

```diff
@@ -2149,3 +2149,45 @@ BOOST_AUTO_TEST_CASE(no_going_backward) {
     BOOST_CHECK_EQUAL(res[0].items[0].departure, time_from_string("2015-01-01 08:10:00"));
     BOOST_CHECK_EQUAL(res[0].items[0].arrival, time_from_string("2015-01-01 08:20:00"));
 }
+
+// 1: A--->B
+// 2:      B---->C
+// We are at A, and we can walk to B, but can't take 2.
```

Comment:
Comment not very clear to me : 
1 and 2 are VJ, and if we walk to B we can't take VJ 2?